### ISSUE TITLE
can not download pretrained model 

### ISSUE THREAD

**Issue by angela000** [chichilicious/Generalized-Zero-Shot-Learning-via-Aligned-Variational-Autoencoders#3]
Hi, thank you for your code very much !  I came across an issue that the pretrained model for various datasets  can not be downloaded, will you please help me fix it ?
best wishes!

**Comment by nmadapan** [comment]
Yes. I have similar problem. It would be great if you can help address this problem. Thanks.
